### PR TITLE
using ECR instead of DockerHub

### PR DESCRIPTION
**Pull request by christina-roperto**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,14 +12,14 @@ plan:
 
 _plan:
 	bash -x scripts/plan_ecr.sh
-	bash -x scripts/plan_aws.sh
```
Comment (lgothelipe) [inline Makefile]: we need the plan aws on the plan stage before do anything, if we need another process under build lets create another script there

File: scripts/plan_aws.sh [hunk 2]
```diff
@@ -1,3 +1,5 @@
+repo=$(cat ./terraform/aws-ecr/repo.txt | sed -n -e 's/^.*repository_url = //p')
+
 cd /code/terraform
 terraform init
-terraform plan -out aws
+TF_VAR_repository_url="$repo" terraform plan -out aws
```
Comment (lgothelipe) [inline scripts/plan_aws.sh]: we could use terraform data sources to get this info or as you suggested put the aws-ecr inside a module and call terraform -target so under task definition you can get the repo url from the module output

File: Makefile (hunk 1 above)
Comment (christina-roperto) [reply Makefile]: We can only plan aws after applying ECR because we need the ECR url.

File: scripts/plan_aws.sh (hunk 2 above)
Comment (christina-roperto) [reply scripts/plan_aws.sh]: I agree but that's beyond this pull request. We can improve later once we get wordpress working.

**Review by luizfdsilva — APPROVED** [review]
Lgtm

**Pull request merged**